lms/djangoapps/courseware/tests/test_date_summary.py

```diff
@@ -195,15 +195,15 @@ def test_todays_date_time_zone_daylight(self):
         Test today's date block displays correctly during
         daylight savings hours
         """
-        self._today_date_helper('Nov 01, 2015 (01:59 PDT)')
+        self._today_date_helper('Nov 01, 2015 (01:59 America/Los_Angeles)')
```

Comment:
It was previously using the time_zone abbr that Kevin wrote. Will switch it back for the interim.